sdk/eventhubs/azure-messaging-eventhubs/test/ut/eventhubs_admin_client.cpp

```diff
@@ -564,14 +570,14 @@ namespace Azure { namespace Messaging { namespace EventHubs { namespace Test {
     request.SetHeader("Accept", "application/json");
     auto result = m_pipeline->Send(request, context);
 
-    auto& val = result->GetBody();
-    std::string bodyAsString{reinterpret_cast<const char*>(val.data()), val.size()};
-    if (result->GetStatusCode() != Azure::Core::Http::HttpStatusCode::Ok)
+    if (result->GetStatusCode() != Azure::Core::Http::HttpStatusCode::Ok
+        && result->GetStatusCode() != Azure::Core::Http::HttpStatusCode::NoContent)
     {
       throw Azure::Core::RequestFailedException(result);
     }
 
-    Azure::Core::Json::_internal::json jsonOutput = ParseAzureCliOutput(bodyAsString);
+    // There is no expected body on a delete eventhub response.
```

Comment:
The HTTP DELETE verb specifies that there is no body for a DELETE verb.

I believe that the policy for the Azure SDK is that it's not our job to validate the responses from the server unless they cannot be parsed (because they are internally inconsistent).